catkin_ws/src/global_perception/scripts/GlobalFuser.py

```diff
@@ -12,23 +12,29 @@ class GlobalFuser():
     def __init__(self):
         rospy.init_node('global_perception', anonymous=False)
         self.rate = rospy.Rate(NODE_RATE)
-        self.sub = rospy.Subscriber('chatter', String, self.nodeCallback)
-        self.pub = rospy.Publisher('global_fused_bboxes', String, queue_size=10)
-        self.rawBuffer = GlobalBBoxBuffer() # class for recieved bbox maybe we need a double buffer?
-        self.predictions = BBoxPredictor() # class for holding predictions for ID'd bbox
 
+        self.sub_n1 = mf.Subscriber('node1/bounding_boxes', BoundingBoxArray)
```

Comment:
so every time we add a node we have to add one of these lines? is there a feature in message filters that can just catch all topics with a certain prefix and proper message type? otherwise we should clean this up to at least be a global variable that defines how many nodes there are